web/ui/mantine-ui/src/components/NotificationsIcon.tsx

```diff
@@ -0,0 +1,62 @@
+import { ActionIcon, Indicator, Popover, Card, Text, Stack, ScrollArea, Group } from "@mantine/core";
+import { IconBell, IconAlertTriangle, IconNetworkOff } from "@tabler/icons-react";
+import { useNotifications } from '../state/useNotifications';
+import NotificationDate from "./NotificationDate";
+import { actionIconStyle } from "../styles";
+
+const NotificationsIcon = () => {
+  const { notifications, notificationCount, isConnectionError } = useNotifications();
+
+  return (
+    (notificationCount === 0 && !isConnectionError) ? null : (
+      <Indicator
+        color={"red"}
+        size={16}
+        label={isConnectionError ? "!" : notificationCount}
+      >
+        <Popover position="bottom-end" withArrow>
```

Comment:
Could you add `shadow="md"` here? If you compare it to the Settings menu dropdown, it's missing the shadow (see https://github.com/prometheus/prometheus/blob/67caa03dc151eb99e09530d7bce7f672be7cba2f/web/ui/mantine-ui/src/components/SettingsMenu.tsx#L20C42-L20C53).